concept-anchor.md

```diff
@@ -0,0 +1,56 @@
+## Title
+Explaining InnerSource to Management by anchoring it to Agile / DevOps / Lean
+
+## Patlet
+Those wanting to bring InnerSource to organizations have difficulty starting a discussion with management. This can sometimes be made easier by anchoring InnerSource to more well known concepts such as Agile / DevOps / Lean.
+
+## Problem
+Often InnerSource is a term that's novel to organizations, making it often difficult to start a conversation with management about adopting InnerSource.
+
+## Context
+An individual sees an opportunity for InnerSource to make an impact within an organization. They would like to discuss InnerSource with management. Being a relatively traditional and bureaucratic company, it is difficult to broach the subject of InnerSource with management as they are reluctant to engage on a topic that they've neither heard of nor know anything about.
+
+These organizations might have an existing strategy, such as Agile, Lean, or DevOps transformation. To gain mindshare, it might be useful to start with these existing, well known concepts and build InnerSource on top of these concepts.
+
+## Forces
```

Comment:
I think another important force here is term dilution:
- As you write below "there is no fixed definition of DevOps". Similar dilution / non-shared understanding of what "Agile" might entail is now the case. Binding something new to a concept already more or less in a "whatever" state might lead to negative impact on the new concept.
- InnerSource needs both bottom-up credibility as well as support from the top or the person introducing this will be at risk of "oh so this is yet another Agile being pushed down our throats" which usually is insta-death of all efforts.
- Introducing the adverse forces here are likely also very helpful to be able to prepare for them.